shared-bindings/displayio/Bitmap.c

```diff
@@ -172,6 +172,115 @@ STATIC mp_obj_t bitmap_subscr(mp_obj_t self_in, mp_obj_t index_obj, mp_obj_t val
     return mp_const_none;
 }
 
+//|     def blit(self, x: int, y: int, source_bitmap: bitmap, x1: int, y1: int, x2: int, y2: int, skip_index: int) -> Any:
+//|         """Inserts the source_bitmap region defined by rectangular boundaries
+//|         (x1,y1) and (x2,y2) into the bitmap at the specified (x,y) location.
+//|         :param int x: Horizontal pixel location in bitmap where source_bitmap upper-left
+//|             corner will be placed
+//|         :param int y: Vertical pixel location in bitmap where source_bitmap upper-left
+//|             corner will be placed
+//|         :param bitmap source_bitmap: Source bitmap that contains the graphical region to be copied
+//|         : param int x1: Minimum x-value for rectangular bounding box to be copied from the source bitmap
+//|         : param int y1: Minimum y-value for rectangular bounding box to be copied from the source bitmap
+//|         : param int x2: Maximum x-value for rectangular bounding box to be copied from the source bitmap
+//|         : param int y2: Maximum y-value for rectangular bounding box to be copied from the source bitmap
```

Comment:
Are the maximum values inclusive or exclusive. I think they are typically exclusive.

Reply:
Please ensure the max values are exclusive below and comment they are here.